addon/services/gravatar.js

```diff
@@ -8,19 +8,20 @@ const {
 
 
 export default Service.extend({
-  hasGravatar(email, secure) {
+  hasGravatar(email, secure, gravatar_host) {
```

Comment:
Can we camelCase this?: `gravatarHost`

Reply:
Sure thing. So used to doing_this, I didn't think to.

Reply:
Done!